boletin-de-pediatria.csl

```diff
@@ -18,109 +18,140 @@
-        </if>
-      </choose>
-    </group>
+    <choose>
+      <!--discard publisher info for articles-->
+      <if type="article-journal article-magazine article-newspaper" match="none">
+        <group delimiter=": " suffix=";">
+          <choose>
+            <if type="thesis">
+              <text variable="publisher-place" prefix="[" suffix="]"/>
+            </if>
+            <else-if type="speech"/>
+            <else>
+              <text variable="publisher-place"/>
+            </else>
+          </choose>
+          <text variable="publisher"/>
+        </group>
+      </if>
+    </choose>
   </macro>
   <macro name="access">
     <choose>
-      <if variable="page" match="none">
-        <choose>
-          <if variable="DOI" match="any">
-            <group prefix=". " delimiter=". ">
-              <date variable="issued" form="text" prefix="Epub ahead of print "/>
-              <text variable="DOI" prefix="DOI: "/>
-            </group>
-          </if>
-          <else>
-            <group delimiter=" ">
-              <text variable="URL"/>
-              <text macro="accessed-date"/>
-            </group>
-          </else>
-        </choose>
+      <if variable="URL">
+        <group delimiter=": ">
+          <text term="available at" text-case="capitalize-first"/>
+          <text variable="URL"/>
+        </group>
       </if>
     </choose>
   </macro>
   <macro name="accessed-date">
     <choose>
       <if variable="URL">
-        <group prefix="(" suffix=")" delimiter=", ">
-          <text macro="date"/>
-          <group delimiter=" ">
-            <text term="accessed"/>
-            <date variable="accessed" form="text"/>
-          </group>
+        <group prefix="[" suffix="]" delimiter=" ">
+          <text term="cited" text-case="lowercase"/>
+          <date variable="accessed" form="text"/>
         </group>
       </if>
     </choose>
   </macro>
   <macro name="container-title">
     <choose>
-      <if type="article-journal" match="any">
-        <text variable="container-title" form="short" strip-periods="true" font-style="normal"/>
+      <if type="article-journal article-magazine chapter paper-conference article-newspaper review review-book entry-dictionary entry-encyclopedia" match="any">
+        <group suffix="" delimiter=" ">
+          <choose>
+            <if type="article-journal review review-book" match="any">
+              <text variable="container-title" form="short" strip-periods="true"/>
+            </if>
+            <else>
+              <text variable="container-title" strip-periods="true"/>
+            </else>
+          </choose>
+          <choose>
+            <if variable="URL">
+              <text term="internet" prefix="[" suffix="]" text-case="capitalize-first"/>
+            </if>
+          </choose>
+        </group>
+        <text macro="edition" prefix=" "/>
       </if>
+      <!--add event-name and event-place once they become available-->
```

Comment:
"event" and "event-place" are available. Can you add these as necessary and remove the note?

Reply:
I'll remove de note, as the are not necessary for this citation style.

Reply:
Okay, please delete this comment from the style then

Reply:
OK. I've sent a new commit.